rpc/src/v1/traits/eth.rs

```diff
@@ -238,3 +385,386 @@ pub trait EthSigning: Sized + Send + Sync + 'static {
 		delegate
 	}
 }
+
+impl<T: Eth> EthRpc for T {
+	fn protocol_version(&self, params: Params) -> Result<Value, Error> {
+		try!(self.active());
+		try!(expect_no_params(params));
+
+		let version = try!(Eth::protocol_version(self));
+
+		Ok(Value::String(format!("{}", version)))
+	}
+
+	fn syncing(&self, params: Params) -> Result<Value, Error> {
+		try!(self.active());
+		try!(expect_no_params(params));
+
+		Eth::syncing(self).map(to_value)
```

Comment:
Wouldn't it be better to have different names instead?

`EthRpc::rpc_syncing` and call would be just `self.syncing()`?